app/Model/Vote/VotingResult.php

```diff
@@ -4,70 +4,146 @@
+    /** @return Candidate[] */
+    public function getNacelni() : array
     {
-        return $this->yesCount === 0 ? 0 : round(($this->yesCount / $this->totalCountOfDelegates)*100, 2);
+        return $this->nacelni;
     }
 
-    public function getNoCount() : int
+    /** @return Candidate[] */
+    public function getNacelnictvoMale() : array
     {
-        return $this->noCount;
+        return $this->prepareNacelnictvo(SkautisCandidate::SEX_MALE);
     }
 
-    public function getAbstainCount() : int
+    /** @return Candidate[] */
+    public function getNacelnictvoFemale() : array
     {
-        return $this->abstainCount + $this->getNotVotedCount();
+        return $this->prepareNacelnictvo(SkautisCandidate::SEX_FEMALE);
     }
 
-    public function getNotVotedCount() : int
+    /** @return Candidate[] */
+    private function prepareNacelnictvo(string $sex) : array
     {
-        return $this->totalCountOfDelegates - ($this->yesCount + $this->noCount + $this->abstainCount);
+        $personIds = $this->getNovyNacelniciIds();
+        $res       = [];
+        $append    = [];
+        foreach ($this->nacelnictvo as $i => $n) {
+            if ($n->getSex() !== $sex) {
+                continue;
+            }
+
+            if (in_array($n->getPersonId(), $personIds)) {
+                $n->setResultVotingNote($n->getSex() === SkautisCandidate::SEX_FEMALE ? 'Zvolena (místo)náčelní' : 'Zvolen (místo)náčelníkem');
+                $append[] = $n;
+            } else {
+                $res[] = $n;
+            }
+        }
+
+        return array_merge($res, $append);
     }
 
-    public function getTotalVotingCount() : int
+    /** @return Candidate[] */
+    public function getUrkj() : array
     {
-        return $this->yesCount + $this->noCount;
+        return $this->urkj;
     }
 
-    public function getCountOfVotes() : int
+    /** @return Candidate[] */
+    public function getRsrj() : array
     {
-        return $this->yesCount + $this->noCount + $this->abstainCount;
+        return $this->rsrj;
     }
 
-    public function getMinVotes() : int
+    public function getCountOfDelegates() : int
     {
-        return (int) ceil($this->totalCountOfDelegates * (3/5));
+        return $this->countOfDelegates;
     }
 
-    public function getResult() : Choice
+    /** @return int[] */
+    public function getNovyNacelniciIds() : array
     {
-        if ($this->getYesCount() >= $this->getMinVotes()) {
-            return Choice::YES();
-        }
+        $nacelni  = $this->getNacelni()[0];
+        $nacelnik = $this->getNacelnik()[0];
 
-        return Choice::NO();
+        return [
+            $nacelni->getPersonId(),
+            $nacelni->getRunningMate()->getPersonId(),
+            $nacelnik->getPersonId(),
+            $nacelnik->getRunningMate()->getPersonId(),
+        ];
     }
+
+    public function isQuorumSatisfied() : bool
+    {
+        return $this->countOfVotedDelegates > ($this->countOfDelegates/2);
```

Comment:
podle zadání by mělo stačit 50 %, tedy >=

Reply:
teď to vlastně pořád není dobře kvůli celočíselnému dělení, předělal bych to na 

`return $this->countOfVotedDelegates * 2 >= $this->countOfDelegates;`